ffmpeg/ffmpeg.go

```diff
@@ -269,6 +273,16 @@ func (t *Transcoder) Transcode(input *TranscodeOptionsIn, ps []TranscodeOptions)
 	if err != nil {
 		return nil, err
 	}
+	for _, p := range ps {
+		if (p.From > 0 || p.To > 0) && (p.VideoEncoder.Name == "drop" || p.VideoEncoder.Name == "copy") {
```

Comment:
```suggestion
		if (p.To < p.From || p.From < 0){
		        glog.Warning("'To' should be after 'From' and should be positive integer")
			return nil, ErrTranscoderClipConfig
		}
		if (p.VideoEncoder.Name == "drop" || p.VideoEncoder.Name == "copy") {
		        glog.Warning("Could clip only when transcoding video")
			return nil, ErrTranscoderClipConfig
```
I think the above suggestion works as well because the following parameters could be passed under the here conditions.
  from -1 to -2, from 0 to -3, from -3 to 2

Reply:
thanks for catching that up!
fixed in fc2eb738651f2238bd98c48f4de73004e8c6b342